GeoLib/OctTree.h

```diff
@@ -86,6 +86,8 @@ class OctTree {
     /// private constructor
     /// @param ll lower left point
     /// @param ur upper right point
+    /// @param eps the euclidean distance as a threshold to make objects unique
+    /// [default std::numeric_limits<double>::epsilon()]
```

Comment:
Where is the default set? The method declaration does not declare a default value.